.gitignore

```diff
@@ -1,5 +1,5 @@
-*swp
-node_modules
-*.log
-.DS_Store
-.idea
+*swp
+node_modules
+*.log
+.DS_Store
+.idea
```

Comment:
Hopefully caught all these in my latest PR. Will pay attention to this going forward.